webapp/components/wpt-metadata.js

```diff
@@ -170,13 +184,39 @@ class WPTMetadata extends PathInfo(LoadingState(PolymerElement)) {
     url.searchParams.set('includeTestLevel', true);
     url.searchParams.set('products', productVal.join(','));
     this.load(
-      window.fetch(url).then(r => r.json()).then(metadata => {
-        this.metadata = metadata;
+      window.fetch(url).then(r => r.json()).then(mergedMetadata => {
+        this.mergedMetadata = mergedMetadata;
       })
     );
   }
 
-  computTestResultSet(searchResults) {
+  // loadPendingMetadata is called when wpt-metadata.js is initialized
+  // through constructor() or when users triage new metadata, unlike loadMergedMetadata().
+  loadPendingMetadata() {
+    const url = new URL('/api/metadata/pending', window.location);
```

Comment:
This appears to only be for turning a relative URL into an absolute one, but `fetch(url)` already resolves relative URLs. Does just `fetch('/api/metadata/pending')` work?

Reply:
You are right. `fetch will load the path relative to the url you have opened if you don't begin with / or specify absolute url.` However, all other instances of `fetch` in this code base uses absolute path. Lets keep it that way for consistency.